tasks/collect-mrrc-params/collect-mrrc-params.yaml

```diff
@@ -0,0 +1,47 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: collect-mrrc-params
+spec:
+  params:
+    - name: dataJsonPath
+      type: string
+      description: path to data json file
+  results:
+    - name: mrrcParamFilePath
+      description: path of the env file for following tasks
+    - name: cosignPubKeyConfig
+      description: the configmap name for cosign verify public key
+    - name: charonAWSSecret
+      description: the secret name for charon aws credential file
+  steps:
+    - name: collect-mrrc-params
+      image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
+      script: |
+        set -eux
+
+        MRRC_ENV_FILE_PATH="$(dirname $(params.dataJsonPath))/mrrc.env"
+
+        environment="$(jq -jer '.mrrc.environment' $(params.dataJsonPath))"
+        release="$(jq -jer '.mrrc.release' $(params.dataJsonPath))"
+        target="$environment-maven-$release"
+        printf "\nexport MRRC_TARGET=$target" >> "$MRRC_ENV_FILE_PATH"
+
+        productName="$(jq -jer '.mrrc.product.name' $(params.dataJsonPath))"
+        productVersion="$(jq -jer '.mrrc.product.version' $(params.dataJsonPath))"
+        printf "\nexport MRRC_PRODUCT_NAME=$productName" >> "$MRRC_ENV_FILE_PATH"
+        printf "\nexport MRRC_PRODUCT_VERSION=$productVersion" >> "$MRRC_ENV_FILE_PATH"
+
+        zipRegistry="$(jq -jer '.mrrc.product.zipRegistry' $(params.dataJsonPath))"
+        printf "\nexport MRRC_ZIP_REGISTRY=$zipRegistry" >> "$MRRC_ENV_FILE_PATH"
+
+        jq -jer '.mrrc.awsSecret' $(params.dataJsonPath) > "$(results.charonAWSSecret.path)"
+
+        cosignPubKey="$(jq -jr '.mrrc.product.cosignPubKeyConfig // "cosign-pub-key"' $(params.dataJsonPath))"
+        printf "$cosignPubKey" > "$(results.cosignPubKeyConfig.path)"
+
+        printf "$MRRC_ENV_FILE_PATH" > "$(results.mrrcParamFilePath.path)"
+      workingDir: $(workspaces.data.path)
```

Comment:
hmm I don't think we need this line. I don't see it in the other tasks and they do very similar things

Reply:
Removed